docs/ref/settings.txt

```diff
@@ -1409,6 +1409,13 @@ If not ``None``, this will be used as the value of the ``SCRIPT_NAME``
 environment variable in any HTTP request. This setting can be used to override
 the server-provided value of ``SCRIPT_NAME``, which may be a rewritten version
 of the preferred value or not supplied at all.
+It is also used to set the URL resolver script prefix outside of request
+contexts (management commands, standalone scripts) to generate correct URLs when
```

Comment:
outside of the request/response cycle (e.g. in management commands and standalone scripts)